src/binary-finder.ts

```diff
@@ -13,28 +13,7 @@ import { downloadBiome, getDownloadedVersion } from "./downloader";
 import { debug, info } from "./logger";
 import { fileExists, hasNodeDependencies } from "./utils";
 
-export type LocatorStrategy = {
-	/**
-	 * Name of the strategy.
-	 */
-	name: string;
-
-	/**
-	 * Finds the `biome` binary using the locator's strategy.
-	 *
-	 * When passing a project, the locator should use the given path as the
-	 * context for finding the binary.
-	 */
-	find: (path?: Uri) => Uri | undefined | Promise<Uri | undefined>;
-};
-
-export type BinaryFinderResult = Promise<
-	| {
-			bin: Uri;
-			strategy: LocatorStrategy;
-	  }
-	| undefined
->;
```

Comment:
I removed the `LocatorStrategy` because it wasn't typesafe. The path was sometimes required, and sometimes not. Fortunately we didn't actually do anything with the `strategy` as returned from `BinaryFinderResult`, so it was safe to remove.

Reply:
Yep, a remnant that I forgot to remove when I stopped using it. Thanks!